.happy/terraform/envs/dev/main.tf

```diff
@@ -19,6 +19,6 @@ module stack {
   backend_memory               = 8192
```

Comment:
we should make this `8192 * backend_cpus`

Reply:
I don't think this is valid for all values of backend_cpus. Check here for valid configs:
https://aws.amazon.com/fargate/pricing/

Reply:
Ah, that makes more sense, why we need to have these asymmetric configurations.